### PR TITLE
Add UG Sort Command

### LINKED ISSUE #90: Update User Guide to include new commands
Update the User Guide to include new commands like Sort Command for user readability

### PR DESCRIPTION
**Pull request by DKHB0207**
Updated the user guide to include Sort Command and description and enabled assertions in build.gradle.
resolves #90

### CODE REVIEW

**Review by yockcheng — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -177,6 +208,24 @@ Examples:
...
+- `ORDER` indicates whether the sorted list is **ascending** or **descending**.
+- **Ascending** is represented with integer 1.
+- **Descending** is represented with integer -1.
+- `n/` indicates sorting according to name.
+- `i/` indicates sorting according to student id.
+- `tut/` indicates sorting according to tutorial attendance.
+
+Examples:
+
+- `sort -1 i/` sorts the person list in descending order according to student id.
```
Comment (yockcheng) [inline docs/UserGuide.md:226]: Format for tutorial here is wrong. Should include the tutorial number for sorting too.

**Review by yockcheng — CHANGES_REQUESTED** [review]
Overall, LGTM but there's a small mistake on the tutorial format for sorting.

File: docs/UserGuide.md (hunk 1 above)
Comment (DKHB0207) [reply docs/UserGuide.md:226]: Oh my bad! Will amend immediately.

**Review by yockcheng — APPROVED** [review]
LGTM!

**Pull request merged**